free-programming-books-zh.md

```diff
@@ -247,8 +247,8 @@
 
 ### 编程艺术
 
-* [取悦的工序：如何理解游戏](http://read.douban.com/ebook/4972883/) (豆瓣阅读，免费书籍)
-* [每个程序员都应该了解的内存知识(译)](http://www.oschina.net/translate/what-every-programmer-should-know-about-memory-part1?print)【第一部分】
+* [取悦的工序：如何理解游戏](http://read.douban.com/ebook/4972883/) (豆瓣阅读，免费书籍)<!--死链，等待删除-->
```

Comment:
Maybe we could just remove it.
Or submit another PR to remove it, but not and something we will remove the line in the future.

Reply:
I will open a new PR to remove those books that is referenced to dead link.